From 9341293e71c03fe606edc9157bf1e13e3dd5b507 Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Fri, 17 Dec 2010 11:35:38 +0100
Subject: [PATCH] socket extconf: hardcode wide-getaddr info test outcome to true

Without this the socket extension doesn't build correctly
Refreshed for ruby by Tomas Frydrych

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>
Signed-off-by: Tomas Frydrych <tomas@tomtain.com>
---
 ext/socket/extconf.rb |    5 +++++
 1 file changed, 5 insertions(+)

Index: ruby-2.2.1/ext/socket/extconf.rb
===================================================================
--- ruby-2.2.1.orig/ext/socket/extconf.rb
+++ ruby-2.2.1/ext/socket/extconf.rb
@@ -593,6 +593,11 @@ EOS
     raise "unexpected enable_config() value"
   end
 
++
++# Ignore the actual result of the above test and assume that
++# everything is OK.
++getaddr_info_ok = true
++
   if ipv6 and not getaddr_info_ok
     abort <<EOS
 
